packages/search/src/ago/get-items.ts

```diff
@@ -11,14 +12,47 @@ export async function getItems(
   authentication?: UserSession
 ): Promise<ISearchResult<IItem>> {
   const agoParams = encodeAgoQuery(params);
-  return searchItems({
-    ...agoParams,
-    params: {
-      token,
-      countFields: agoParams.countFields,
-      countSize: agoParams.countSize
-    },
-    portal,
-    authentication
-  });
+  if (agoParams.countFields) {
+    // if countFields are defined
+    // AGO allows only 3 countFields at max
+    const chunkedCountFields = chunkArray(
+      agoParams.countFields.split(","),
+      3
```

Comment:
i'd define as a constant `const MAX_COUNTFIELDS = 3` and then delete the comment